packages/api/src/command/medical/document/document-query.ts

```diff
@@ -42,14 +43,14 @@ export async function queryDocumentsAcrossHIEs({
   facilityId,
   override,
   cxDocumentRequestMetadata,
-  skipDocQueryStatusCheck = false,
+  forceQuery = false,
```

Comment:
OOC, this is not part of this PR, why showing up as a change here? Did you rebase your branch from `develop`? It seems like a bad merge, since the previous version already had `forceQuery`.

Not a huge deal, mostly so you know and watch out for those, its fairly easy to introduce bugs when merging.

Reply:
I merged with develop since I had conflicts. Is this change a bug?

Reply:
No, like I said this is not a big deal, its mostly so you're aware.

The fact you merged/rebased and this line shows up as changed means that there was some mistake in the merging - this line is already in develop as you can see [here](https://github.com/metriport/metriport/blob/2b0249eb5fc4c97c211bc9616c84583f61e9f3cb/packages/api/src/command/medical/document/document-query.ts#L44).

Mostly sharing this so you're aware for next time. 👍